app/webpacker/styles/_custom.scss

```diff
@@ -5,3 +5,16 @@
 .govuk-green {
   color: govuk-colour("green");
 }
+
+.app-contents {
```

Comment:
This makes the options summary list look like Matt's prototype:

![Screenshot from 2021-04-13 09-31-57](https://user-images.githubusercontent.com/8156884/114525078-b21b3e00-9c3d-11eb-9716-4c1287e18388.png)